go/datas/remote_database_handlers.go

```diff
@@ -97,7 +98,7 @@ func createHandler(hndlr Handler, versionCheck bool) Handler {
 		w.Header().Set(NomsVersionHeader, constants.NomsVersion)
 
 		if versionCheck && req.Header.Get(NomsVersionHeader) != constants.NomsVersion {
-			verbose.Log("Returning version mismatch error")
+			log.Printf("returning version mismatch error")
```

Comment:
...and then here you changed AWAY from `verbose.Log` to use `log.Printf` directly. What's the right thing for code to do?